sdk/servicebus/test-resources.json

```diff
@@ -68,7 +71,7 @@
   "resources": [
     {
       "type": "Microsoft.ServiceBus/namespaces",
-      "apiVersion": "2021-05-01",
+      "apiVersion": "2021-01-01-preview",
```

Comment:
Is there a reason we're using a different API version that is older?

Reply:
When using 2021-05-01 api version, failed to deploy test resource in weekly pipeline. More details please check [issue#26058](https://github.com/Azure/azure-sdk-for-java/issues/26058).